pkg/apis/kops/cluster.go

```diff
@@ -143,13 +143,21 @@ type ClusterSpec struct {
 	// This API component is under construction, will remove this comment
 	// once this API is fully functional.
 	Assets *Assets `json:"assets,omitempty"`
+
+	// IAM field adds control over the IAM security policies applied to resources
+	IAM *IAMSpec `json:"iam,omitempty"`
 }
 
 type Assets struct {
 	ContainerRegistry *string `json:"containerRegistry,omitempty"`
 	FileRepository    *string `json:"fileRepository,omitempty"`
 }
 
+// IAMSpec adds control over the IAM security policies applied to resources
+type IAMSpec struct {
+	Legacy bool `json:"legacy"`
```

Comment:
My preference is to include `omitempty` here (`false` is considered empty), but we can add that later - that's not breaking (I believe)